gp-templates/translation-row-editor-meta.php

```diff
@@ -53,113 +53,118 @@
-		</dl>
-	<?php endif; ?>
-	<?php references( $project, $translation ); ?>
+	$meta_sidebar .= '			</dt>';
+	$meta_sidebar .= '			<dd>' . gp_link_user_get( $translation->user_last_modified ) . '</dd>';
+	$meta_sidebar .= '		</dl>';
+}
+ob_start();
+references( $project, $translation );
+$meta_sidebar .= ob_get_clean();
+
+$meta_sidebar .= '	<dl>';
+$meta_sidebar .= '		<dt>' . __( 'Priority:', 'glotpress' ) . '</dt>';
+if ( $can_write ) {
+	$meta_sidebar .= '			<dd>';
+	$meta_sidebar .= gp_select(
+		'priority-' . $translation->original_id,
+		GP::$original->get_static( 'priorities' ),
+		$translation->priority,
+		array(
+			'class'      => 'priority',
+			'tabindex'   => '-1',
+			'data-nonce' => wp_create_nonce( 'set-priority_' . $translation->original_id ),
+		)
+	);
+	$meta_sidebar .= '			</dd>';
+	} else {
+	$meta_sidebar .= '			<dd>';
+	$meta_sidebar .= esc_html(
+		gp_array_get(
+			GP::$original->get_static( 'priorities' ),
+			$translation->priority,
+			_x( 'Unknown', 'priority', 'glotpress' )
+		)
+	);
+	$meta_sidebar .= '			</dd>';
+}
+$meta_sidebar .= '	</dl>';
+
+$meta_sidebar .= '	<dl>';
+$meta_sidebar .= '		<dt>' . __( 'More links:', 'glotpress' );
+$meta_sidebar .= '			<ul>';
+				foreach ( $more_links as $more_link ) {
+					$meta_sidebar .= '					<li>';
+					$meta_sidebar .= $more_link;
+					$meta_sidebar .= '					</li>';
+				}
+$meta_sidebar .= '			</ul>';
+$meta_sidebar .= '		</dt>';
+$meta_sidebar .= '	</dl>';
+$meta_sidebar .= '</div>';
 
-	<dl>
-		<dt><?php _e( 'Priority:', 'glotpress' ); ?></dt>
-		<?php if ( $can_write ) : ?>
-			<dd>
-				<?php
-				echo gp_select(
-					'priority-' . $translation->original_id,
-					GP::$original->get_static( 'priorities' ),
-					$translation->priority,
-					array(
-						'class'      => 'priority',
-						'tabindex'   => '-1',
-						'data-nonce' => wp_create_nonce( 'set-priority_' . $translation->original_id ),
-					)
-				);
-				?>
-			</dd>
-		<?php else : ?>
-			<dd>
-				<?php
-				echo esc_html(
-					gp_array_get(
-						GP::$original->get_static( 'priorities' ),
-						$translation->priority,
-						_x( 'Unknown', 'priority', 'glotpress' )
-					)
-				);
-				?>
-			</dd>
-		<?php endif; ?>
-	</dl>
+$defined_vars = get_defined_vars();
+
+/**
+ * Filter the content in the sidebar.
+ *
+ * @since 4.0.0
+ *
+ * @param string $meta_sidebar Default content for the sidebar.
+ * @param array  $defined_vars The defined vars.
+ */
+$meta_sidebar = apply_filters( 'gp_right_sidebar', $meta_sidebar, $defined_vars );
 
-	<dl>
-		<dt><?php _e( 'More links:', 'glotpress' ); ?>
-			<ul>
-				<?php foreach ( $more_links as $more_link ) : ?>
-					<li>
-						<?php
-						// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
-						echo $more_link;
-						?>
-					</li>
-				<?php endforeach; ?>
-			</ul>
-		</dt>
-	</dl>
-</div>
+// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+echo $meta_sidebar;
```

Comment:
Could we escape this output as well? The list of tags will be long but we'd be protected against mischief by the filter.

Reply:
Done [here](https://github.com/GlotPress/GlotPress/pull/1531/commits/2ccb9cba87fc0929054b393c13e002324af9251c).